### DISCUSSION TITLE
Block tabs running in the background

### DISCUSSION THREAD

**Discussion by kitty-ku** [proginosko/LeechBlockNG#406] [Q&A] [answered]
How do I set up the timer to keep ticking when the limited tab is open in the background? Can this even be done?
Neither the FAQ or the Docs have 'background' or 'unfocus' as keywords.

**Comment by proginosko** [comment, marked as answer]
Deselect the option "Count time spent on these sites only when browser tab is active" under "When to Block".

**Comment by lubokkanev** [comment]
Is it possible to count time when **browser tab** is active, but also count when the **browser** itself isn't active? So counting when the youtube tab is active and I'm watching, but I'm clicking on a window of another app.

**Comment by proginosko** [comment]
> Is it possible to count time when **browser tab** is active, but also count when the **browser** itself isn't active? So counting when the youtube tab is active and I'm watching, but I'm clicking on a window of another app.

Select the option "Treat browser windows as always focused" under General / Miscellaneous.

**Comment by lubokkanev** [comment]
> Select the option "Treat browser windows as always focused" under General / Miscellaneous.

I tried using this option but it seems there's a bug. When I have multiple tabs open, the timer skips time. 
For example, I have a rule for youtube - max 70 minutes per day. If I open 5 youtube tabs and watch 2 minutes on each, that's 10 minutes total, but my timer is left with only like 20 minutes, instead of 60. 
Maybe I should open a separate issue?

**Reply by proginosko** [reply]
Yes, this should probably be opened as a new issue.

Are all the tabs in the same browser window? Which browser are you using?

**Reply by lubokkanev** [reply]
Same window. Firefox.

**Reply by proginosko** [reply]
And this happens only when "Treat browser windows as always focused" is selected?

**Reply by lubokkanev** [reply]
Yes. Disabling it fixes it.

**Reply by proginosko** [reply]
Weird! That isn't happening on my setup. Can you open this as a new issue, with a minimal-case series of steps to reproduce?
